.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: go
+go_import_path: github.com/deislabs/secrets-store-csi-driver
+dist: bionic
+
+env:
+  global:
+    - GO111MODULE=on
+
+go:
+  - "1.12.1"
+
+jobs:
+  include:
+    - stage: test
+      name: "build test"
+      install:
+        - GO111MODULE=off go get github.com/mattn/goveralls
+      script:
+        - make build
+        - go test -covermode=count -coverprofile=profile.cov ./pkg/...
+    - stage: test
+      name: "unit test"
+      script:
+        - make test
+    - stage: test
+      name: "sanity test"
+      install:
+        - sudo apt update && sudo apt install cifs-utils procps -y
+      script: make sanity-test
```

Comment:
Let's comment this out for now until we have travis ci setup.

Reply:
@ritazh I've removed the travis config and the `sanity-test` job in circleci. Will add the travis config back once the travis CI is setup. But otherwise the PR is ready to be merged.